src/language-handlebars/printer-glimmer.js

```diff
@@ -267,9 +267,21 @@ function print(path, options, print) {
         text = "\\" + text;
       }
 
+      const trimmedValue = text
+        .replace(/^\s+/g, leadingSpace)
+        .replace(/\s+$/, trailingSpace);
+
+      // Glimmer parser treats attribute values with mustaches differently
```

Comment:
This comment might removed - it doesn't explain how prettier printer works (but the undelying ast).